ontology/observable/observable.ttl

```diff
@@ -7843,17 +7843,41 @@ observable:WirelessNetworkConnectionFacet
 	rdfs:label "WirelessNetworkConnectionFacet"@en ;
 	rdfs:comment "A wireless network connection facet is a grouping of characteristics unique to a connection (completed or attempted) across an IEEE 802.11 standards-conformant digital network (a group of two or more computer systems linked together). [based on https://www.webopedia.com/TERM/N/network.html]"@en ;
 	sh:property
+		[
+			sh:datatype xsd:dateTime ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path observable:endTime ;
```

Comment:
Update: Please remove `endTime` and `startTime` from the added properties.  They are present in `NetworkConnectionFacet`, so the need of end/start time is already met.

Reply:
Removed properties